networkx/readwrite/multiline_adjlist.py

```diff
@@ -246,7 +246,7 @@ def parse_multiline_adjlist(
         if not line:
             continue
         try:
-            (u, deg) = line.strip().split(delimiter)
+            (u, deg) = line.rstrip("\n").split(delimiter)
```

Comment:
I'm not sure we should treat this the same way. `deg` has to be there for this to work. It can't be empty or whitespace. So unless I'm missing something we should never have trouble with a tab delimiter that has only whitespace after it -- or at least that line should raise an error.